src/main/java/entity/room/Room.java

```diff
@@ -0,0 +1,28 @@
+package entity.room;
+
+import java.util.List;
+
+import entity.player.Player;
+
+/**
+ * The representation of a game room in our program.
+ */
+public interface Room {
+    /**
+     * Returns the status of the room: "Initializing", "In Progress", or "Over".
+     * @return the status of the room.
+     */
+    String getStatus();
```

Comment:
I don't foresee any use cases for status of a room because our program will only ever have one active room at a time. What might be more useful to know which player's turn it is!

Reply:
Okay, I see your point.

Reply:
Hmm... I wonder if a status parameter is even needed at all...